tests/robot-tests/tests/admin_and_public_2/bau/publish_release_and_amend.robot

```diff
@@ -170,6 +170,45 @@ Add embedded dashboard to third accordion section
     user waits until h1 is visible    DfE Analytical Services R-Shiny data dashboard template (h1)    90
     unselect frame
 
+User navigates to Data blocks page to edit block
+    [Documentation]    EES-3009
+    user clicks link    Data blocks
+    user waits until h2 is visible    Data blocks    %{WAIT_SMALL}
+
+Edit data block
+    #just updating the table title and source
+    user waits until table is visible
+    user clicks link    Edit block    css:tbody > tr:first-child
+    user waits until h2 is visible    ${DATABLOCK_NAME}
+    user waits until h2 is visible    Data block details
+
+    user enters text into element    id:dataBlockDetailsForm-name    ${DATABLOCK_NAME}
+    user clears element text    id:dataBlockDetailsForm-heading
```

Comment:
I don't think you need to use `user clears element text` here (and in other lines added in this PR), as the `user enters text into element` keyword also contains it.

From `common.robot`:

```
user enters text into element
    [Arguments]    ${selector}    ${text}
    user sets focus to element    ${selector}
    user waits until element is visible    ${selector}    %{WAIT_SMALL}
    user clears element text    ${selector}
    press keys    ${selector}    ${text}
    sleep    0.3
```